kolibri/plugins/user_profile/assets/src/views/ChangeFacility/SelectFacility.vue

```diff
@@ -0,0 +1,279 @@
+<template>
+
+  <div>
+    <span class="headercontainer">
+      <h1>{{ $tr('documentTitle') }}</h1>
+
+      <transition name="spinner-fade">
+
+        <div v-if="discoveringPeers">
+          <KLabeledIcon>
+            <template #icon>
+              <KCircularLoader :size="16" :stroke="6" class="loader" />
```

Comment:
`"loader"` class doesn't seem to be used

Reply:
yeah, good catch